src/components/LoanCards/RecommendedLoanCard.vue

```diff
@@ -0,0 +1,486 @@
+		basket {
+			# for isInBasket
+			items {
+				values {
+					id
+				}
+			}
+		}
+	}
+	lend {
+		loan(id: $loanId) {
+			id
+			geocode {
+				country {
+					name
+					isoCode
+				}
+			}
+			image {
+				default: url(customSize: "w480h300")
+				retina: url(customSize: "w960h600")
+			}
+			name
+			sector {
+				name
+			}
+			whySpecial
+
+			# for isLentTo
+			userProperties {
+				lentTo
+			}
+
+			# for loan-use-mixin
+			use
+			status
+			loanAmount
+			borrowerCount
+
+			# for percent-raised-mixin
+			loanFundraisingInfo {
+				fundedAmount
+				reservedAmount
+			}
+
+			# for time-left-mixin
+			plannedExpirationDate
+		}
+	}
+}`;
+
+export default {
+	props: {
+		loanId: {
+			type: Number,
+			required: true,
+		}
+	},
+	inject: ['apollo'],
+	mixins: [loanUseMixin, percentRaisedMixin, timeLeftMixin],
+	components: {
+		FundraisingStatusMeter,
+		KvFlag,
+		KvLoadingPlaceholder,
+		KvLoadingParagraph,
+		LendButton,
+		WhySpecial,
+	},
+	data() {
+		return {
+			loan: null,
+			basketItems: null,
+			isLoading: false,
+		};
+	},
+	computed: {
+		borrowerName() {
+			return this.loan?.name || '';
+		},
+		countryISO() {
+			return this.loan?.geocode?.country?.isoCode || '';
+		},
+		countryName() {
+			return this.loan?.geocode?.country?.name || '';
+		},
+		imageRetinaUrl() {
+			return this.loan?.image?.retina || '';
+		},
+		imageUrl() {
+			return this.loan?.image?.default || '';
+		},
+		isInBasket() {
+			if (!Array.isArray(this.basketItems)) {
+				return false;
+			}
+			// eslint-disable-next-line no-underscore-dangle
+			const loanItems = this.basketItems.filter(item => item.__typename === 'LoanReservation');
+			return loanItems.indexOf(this.loanId) > -1;
+		},
+		isLentTo() {
+			return this.loan?.userProperties?.lentTo;
+		},
+		sectorName() {
+			return (this.loan?.sector?.name || '').toLowerCase();
+		},
+		whySpecial() {
+			const text = this.loan?.whySpecial || '';
+			return text.toString().charAt(0).toLowerCase() + text.toString().slice(1);
+		},
+	},
+	methods: {
+		fetchLoanData() {
+			if (!this.loan) {
+				this.isLoading = true;
+			}
+			this.apollo.query({
+				variables: {
+					loanId: this.loanId,
+				},
+				query: loanQuery,
+			}).then(result => {
+				this.isLoading = false;
+				this.loan = result.data?.lend?.loan || null;
+				this.basketItems = result.data?.shop?.basket?.items?.values || null;
+			});
+			// TODO: error handling
+		},
+	},
+	serverPrefetch() {
+		return this.fetchLoanData();
+	},
+	mounted() {
+		this.fetchLoanData();
+		// TODO: use a watch query to catch changes to loan data?
+		// TODO: update variables for query when loan id changes?
+	},
+};
+</script>
+
+<style lang="scss">
+@import 'settings';
+
+%nested-column-flex {
+	display: flex;
+	flex-flow: column nowrap;
+	flex-grow: 1;
+}
+
+$largest: rem-calc(1250);
+
+.rec-loan-card {
+	@extend %nested-column-flex;
+
+	flex-shrink: 0;
+	min-width: rem-calc(230);
+	width: 230/320 * 100%;
+	max-width: rem-calc(280);
```

Comment:
One thought on these loan cards is let the parent handle the widths (remove max-width, set width to always be 100%). We could then drop them into our standard grid use our `small-12 large-6 columns` type syntax do the work.

Reply:
👍 

I initially started with that, but the styles were different enough that I stopped using the row/column classes and just defined my own, but now that I've got it this far I think there's a different way I can achieve the results I'm looking for using the row/column classes.